WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -81,21 +81,18 @@ public void Initialize(NodesCollection nodes, WasabiSynchronizer synchronizer, U
 				.ToProperty(this, x => x.Status)
 				.DisposeWith(Disposables);
 
-			Observable.FromEventPattern<NodeEventArgs>(nodes, nameof(nodes.Added))
+			Observable
+				.Merge(Observable.FromEventPattern<NodeEventArgs>(nodes, nameof(nodes.Added)).Select(x => true)
+				.Merge(Observable.FromEventPattern<NodeEventArgs>(nodes, nameof(nodes.Removed)).Select(x => true)
+				.Merge(Synchronizer.WhenAnyValue(x => x.TorStatus).Select(x => true))))
 				.ObserveOn(RxApp.MainThreadScheduler)
 				.Subscribe(_ =>
 				{
-					SetPeers(Nodes.Count);
+					// Set peers to 0 if Tor is not running, because we get Tor status from backend answer so it seems to the user that peers are connected over clearnet, while they do not.
```

Comment:
```suggestion
					// Set peers to 0 if Tor is not running, because we get Tor status from backend answer so it seems to the user that peers are connected over clearnet, while they are not.
```